native/tools/swig-config/spine.i

```diff
@@ -726,4 +727,11 @@ using namespace spine;
         }
         $self->setFrame(frameIndex, time, spDrawOrder);
     }
+}
+
+%extend spine::Skeleton {
+    void setSkinByName(const std::string &skinName) {
+        spine::String name(skinName.data());
+        $self->setSkin(name);
+    }
```

Comment:
Why not just rename `setSkin` to `setSkinByName` ?

Reply:
I mean
```
%rename(setSkinByName) spine::Skeleton:: setSkin;
```

Reply:
there are 2 setSkin function in spine::Skeleton